functions/exercise13/script.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
+
+function getFactorsOfPositiveInteger(number) {
+    var array = [];
```

Comment:
You should not name variables on the basis of their type - you should choose names that will specify what it may contain. Please consider adding more meaningful name here.